src/command_modules/azure-cli-eventhubs/azure/cli/command_modules/eventhubs/_params.py

```diff
@@ -0,0 +1,119 @@
+            c.argument('capture_interval_seconds', type=int, help='The time window allows you to set the frequency with which the capture to Azure Blobs will happen, value should between 60 to 900 seconds')
+            c.argument('capture_size_limit_bytes', type=int, help='The size window defines the amount of data built up in your Event Hub before an capture operation, value should be between 10485760 to 524288000 bytes')
+            c.argument('destination_name', help='Name for capture destination')
+            c.argument('storage_account_resource_id', help='Resource id of the storage account to be used to create the blobs')
+            c.argument('blob_container', help='Blob container Name')
+            c.argument('archive_name_format', help='Blob naming convention for archive, e.g. {Namespace}/{EventHub}/{PartitionId}/{Year}/{Month}/{Day}/{Hour}/{Minute}/{Second}. Here all the parameters (Namespace,EventHub .. etc) are mandatory irrespective of order')
+
+    # region EventHub Authorizationrule
+    for scope in ['eventhubs eventhub authorization-rule', 'eventhubs eventhub authorization-rule keys list', 'eventhubs eventhub authorization-rule keys renew']:
+        with self.argument_context(scope) as c:
+            c.argument('authorization_rule_name', arg_type=name_type, id_part='child_name_2', help='name of the EventHub AuthorizationRule')
+            c.argument('event_hub_name', id_part='child_name_1', arg_type=event_hub_name_arg_type)
+
+    with self.argument_context('eventhubs eventhub authorization-rule create') as c:
+        c.argument('rights', arg_type=rights_arg_type)
+
+    with self.argument_context('eventhubs eventhub authorization-rule keys renew') as c:
+        c.argument('key_type', arg_type=key_arg_type)
+
+
+# - ConsumerGroup Region
+    with self.argument_context('eventhubs eventhub consumergroup') as c:
+        c.argument('event_hub_name', arg_type=event_hub_name_arg_type)
+        c.argument('consumer_group_name', arg_type=name_type, id_part='child_name_2', help='Name of ConsumerGroup')
+
+    for scope in ['eventhubs eventhub consumergroup create', 'eventhubs eventhub consumergroup update']:
+        with self.argument_context(scope) as c:
+            c.argument('user_metadata', help='Usermetadata is a placeholder to store user-defined string data with maximum length 1024. e.g. it can be used to store descriptive data, such as list of teams and their contact information also user-defined configuration settings can be stored.')
+
+    with self.argument_context('eventhubs eventhub consumergroup list') as c:
+        c.argument('event_hub_name', arg_type=event_hub_name_arg_type)
+
+#   : Region Geo DR Configuration
+    with self.argument_context('eventhubs georecovery-alias') as c:
+        c.argument('alias', id_part='child_name_1', help='Name of the Alias (Disaster Recovery)')
```

Comment:
Using `--alias` instead of `--name` makes sense to me, but consider adding the short option `-a`.

Reply:
implemented